portfolio/src/main/webapp/script.js

```diff
@@ -46,12 +50,35 @@ function changeProject() {
   changeCurrProj = setTimeout("changeProject()", changeProjectTimeMs);
 }
 
+/** Pauses project slideshow on mouse hover */
 function pauseProjectChange() {
   clearTimeout(changeCurrProj);
 }
 
-function fetchMessage() {
-  fetch('/data').then(response => response.json()).then((message) => {
-    document.getElementById('message-container').innerText = message;
+/** Fetches comment(s) and updates the UI to display */
+function fetchCommentAndDisplay() {
+  fetch('/data').then(response => response.json()).then((comments) => {
+    const commentsContainer = document.getElementById('comments-container');
+    comments.forEach((comment) => {
+      const currentCommentContainer = createDivElement();
+      currentCommentContainer.style.padding = "50px 0px";
+      commentsContainer.appendChild(currentCommentContainer);
+      currentCommentContainer.appendChild(createParagraphElement(comment.name));
+      currentCommentContainer.appendChild(createParagraphElement(comment.email));
+      currentCommentContainer.appendChild(createParagraphElement(comment.text));
+    });
   });
+}
+
+/** Creates an <p> element containing text. */
+function createParagraphElement(text) {
+  const paragraphElement = document.createElement('p');
+  paragraphElement.innerText = text;
+  return paragraphElement;
+}
+
+/** Creates an <div> element containing comment object. */
+function createDivElement() {
+  const divElement = document.createElement('div');
+  return divElement;
```

Comment:
Totally optional: You can remove this function because it looks like this function only has one statement, which is to call another function, unless it'll be used often in the future.